script.bash

```diff
@@ -31,33 +31,40 @@ createDirIfNotExist() {
 
 copyConfigurationIfNotExist() {
     if [ ! -s ${DATA_DIR}/${CONFIG} ]; then
-        echo -e "${RED}config.ini file doesn't exist (or it is empty)${DEFAULT}"
+        echo -e "${RED}${CONFIG} file doesn't exist or it is empty. ${GREEN}Creating${DEFAULT}"
         cp ${WORKING_DIR}/${CONFIG_EXAMPLE} ${DATA_DIR}/${CONFIG}
         sudo chmod -R +755 ${DATA_DIR}/${CONFIG}
     fi
 }
 
 copyLoggingIfNotExist() {
     if [ ! -s ${DATA_DIR}/${LOGGING} ]; then
-        echo -e "${RED}logging.ini file doesn't exist (or it is empty)${DEFAULT}"
+        echo -e "${RED}${LOGGING} file doesn't exist or it is empty. ${GREEN}Creating${DEFAULT}"
```

Comment:
`echo -e "${RED}${LOGGING} file doesn't exist or it is empty. ${GREEN}So creating the file...${DEFAULT}"`

Three dots as we have. :D